myfitnesspal/client.py

```diff
@@ -795,3 +801,75 @@ def get_food_item_details(self, mfp_id: int) -> FoodItem:
             serving_sizes=details["serving_sizes"],
             client=self,
         )
+
+    def get_report(
```

Comment:
Several sections here are copy-pasted from `get_measurements()`. Perhaps it would make sense to do some refactoring to reduce code duplication?